client/hud_cl.lua

```diff
@@ -265,3 +265,18 @@ function StopAnim()
 		prop = 0
 	end
 end
+
+CreateThread(function()
+	while true do
+	Citizen.Wait(0)
+	
+		local playerPed = PlayerPedId()
+		local playerVeh = GetVehiclePedIsIn(playerPed, false)

```

Comment:
- Change variable name to `vehicle`.
- Instead of having `PlayerPedId()` as a variable, just add it here so that it looks like `GetVehiclePedIsIn(PlayerPedId(), false)`